gammapy/spectrum/flux_point.py

```diff
@@ -670,3 +670,167 @@ def plot(self, ax=None):
             ax = plt.gca()
 
         # TODO
+
+
+def chi2_flux_points(flux_points_list, models):
```

Comment:
Why pass as list of models and then only access the first model? -> suggest to just pass a model.

Is it really useful to have lists of flux points? -> suggest to create a two-line `FluxPoints.stack` helper function that takes a list of flux points and stacks them, calling `astropy.table.Table.vstack` and to show a docs example how to do that.